src/content/docs/release-notes/docs-release-notes/docs-6-7-2024.mdx

```diff
@@ -0,0 +1,52 @@
+---
+subject: "Docs"
+releaseDate: '2024-06-07'
+version: 'ersion: May 31-June 6, 2024'
+---
+
+### New docs
+
+* A new [OpenTelemetry data in New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview) doc is an index to help you find the OpenTelemetry info you're looking for.
+
+### Major changes
+
+* We've combined all our [.NET API](/docs/apm/agents/net-agent/net-agent-api/net-agent-api) docs so now you can see them all in one place.
+* Consolidated Java and Python APM info into [Monitor APM apps and services with OpenTelemetry](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/apm-monitoring/opentelemetry-apm-intro) so you can get this info in one place.
+
+
+### Minor changes
+
+* Updated [PHP agent operating system support](/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements/#operating-systems) to state that we support an OS version one year after a vendor's end-of-life date. (This saves us some maintenance overhead, because we don't need to list specific versions now.)
+* Our .NET APM agent for .NET Core and Framework now supports `Microsoft.Data.SqlClient` v5.2.1. Also, .NET Framework 4.6.2 or higher requires .NET APM agent version 10.0 or higher.
+* On our homepage, we've broken OpenTelemetry out into its own tile button.
+* In [Forward your logs using the infrastructure agent](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent), clarified that using ANSI encoding on winevtlog addresses empty string logs, but isn't used to convert ANSI to UTF-8. Added example `use-ansi` entries.
+* Updated some non-standard callouts to use our recommended format. 
+* In [Send your logging data with our Log API](/docs/logs/log-api/introduction-log-api/#attributes), added `"msg"` to the JSON message attributes table.
+* Added `http.response.status_code` from http span link to OpenTelemetry docs in [OpenTelemetry APM errors inbox UI](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/apm-monitoring/opentelemetry-apm-ui/#errors-inbox-page).
+* In [APM: Monitor apps and services with OpenTelemetry](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/apm-monitoring/opentelemetry-apm-intro/#apm-agents), added a new section that lists things to keep in mind when you're running APM agents alongside OpenTelemetry tools. 
+
+### Release notes and What's New posts
```

Comment:
@paperclypse I didn't notice this before but this seems to duplicate release notes. I suppose it doesn't hurt.